data/mods/ssb/conditions.ts

```diff
@@ -1194,6 +1206,42 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
 			}
 		},
 	},
+	// Brilliant Condition for Arcticblast
+	brilliant: {
+		name: 'Brilliant',
+		duration: 5,
+		onStart(pokemon) {
+			pokemon.addVolatile("ingrain");
+		},
+		onModifyAtk(atk, pokemon) {
+			return this.chainModify(1.5);
+		},
+		onModifyDef(def, pokemon) {
+			return this.chainModify(1.5);
+		},
+		onModifySpA(spa, pokemon) {
+			return this.chainModify(1.5);
+		},
+		onModifySpD(spd, pokemon) {
+			return this.chainModify(1.5);
+		},
+		onModifySpe(spe, pokemon) {
+			return this.chainModify(1.5);
+		},
+		onUpdate(pokemon) {
+			if (pokemon.status === 'brilliant') {
+				this.add('-activate', pokemon, 'ability: Brilliant');
+				pokemon.removeVolatile('perishsong');
+			}
```

Comment:
this isn't a nonvolatile status?

Reply:
https://github.com/HoeenCoder/Pokemon-Showdown/blob/master/data/moves.ts#L13326